src/integrationTest/resources/extension-config.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+
+    Copyright 2019-present HiveMQ GmbH
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<cloudwatch-extension-configuration>
+    <aws-endpoint-override>http://localstack:4566</aws-endpoint-override>
+    <report-interval>1</report-interval>
+    <zero-value-submission>true</zero-value-submission>
+    <report-raw-count-value>true</report-raw-count-value>
```

Comment:
So if understood this setting correctly the current test would not be influenced by it being true or false. Instead, we would have to count the received metrics and check if a metric with the same value was resend after the report interval, right?